packages/server/src/main/java/software/uncharted/terarium/hmiserver/utils/Messages.java

```diff
@@ -0,0 +1,27 @@
+package software.uncharted.terarium.hmiserver.utils;
+
+import java.util.Locale;
+import org.springframework.context.MessageSource;
+import org.springframework.context.support.MessageSourceAccessor;
+import org.springframework.stereotype.Component;
+
+/**
+ * Helper to simplify accessing i18n messages in code.
+ *
+ * <p>This finds messages automatically found from src/main/resources (files named messages_*.properties)
+ *
+ * <p>For now, we are using a hard-coded English locale.
+ */
+@Component
+public class Messages {
+
+	private final MessageSourceAccessor accessor;
+
+	public Messages(MessageSource messageSource) {
+		this.accessor = new MessageSourceAccessor(messageSource, Locale.ENGLISH);
```

Comment:
How does SpringBoot knows that `messages.properties` is the one to use for english?

Reply:
defaults to message.properties. we can rename to message_en.properties when we want to add a new language. But for now, we only have a default